### PR TITLE
farzad/lesson-1/farzad

### PR DESCRIPTION
**Pull request by farzadabdollahi**

### CODE REVIEW

File: lesson-1/farzad/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="grid">
+      <div class="top-left"></div>
```
Comment (mehrzadnaini) [inline lesson-1/farzad/index.html:8]: Same here `top-left` is a general name which may defined in other toolkits such as bootstrap and you may get some unwanted styles.
One of the best methodologies which you can use is BEM please check following documentation
http://getbem.com/introduction/

File: lesson-1/farzad/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
```
Comment (mehrzadnaini) [inline lesson-1/farzad/index.html:5]: It's better if you use HTML5 meta tags 
you don't need to remember them you should know the reason why we need them and use vscode `html snippets` 
```suggestion
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Logo</title>
        <link rel="stylesheet" href="./styles.css" />
</head>
```

File: lesson-1/farzad/styles.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.grid {
+    display:inline-grid;
+    grid-template-columns:auto auto;
+    gap:5px;
+    width: 10vw;
```
Comment (mehrzadnaini) [inline lesson-1/farzad/styles.css:5]:
```suggestion
    width: 10vw;
    height:10vw
```

File: lesson-1/farzad/styles.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    background-color: rgb(0, 0, 0);
+    border-radius: 0 100px 100px 0;
+    margin-right: 300px;
+    margin-bottom: 200px;
+}
+.bottom-left {
+   
+   width: 65px;
+    height: 34px;
+    /* border-radius: 100%; */
```
Comment (mehrzadnaini) [inline lesson-1/farzad/styles.css:40]:
```suggestion
```

File: lesson-1/farzad/styles.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    width: 10vw;
+}
+
+.top-right {
+    border: 1px solid black;
+    width: 20%;
+    height: 60px;
+    background-color: rgb(0, 0, 0);
+    border-radius: 0 100px 100px 0;
+    margin-top: 200px;
```
Comment (mehrzadnaini) [inline lesson-1/farzad/styles.css:14]: The aim of this task is to create a responsive logo that changes based on the screen width so we should not use `px`

**Review by mehrzadnaini — CHANGES_REQUESTED** [review]

File: lesson-1/farzad/index.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Logo</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="logo">
+      <div class="grid1"></div>
+      <div class="grid2"></div>
+      <div class="grid3"></div>
+      <div class="grid4"></div>
```
Comment (mehrzadnaini) [inline lesson-1/farzad/index.html:15]:
```suggestion
      <div class="logo_grid1"></div>
      <div class="logo_grid2"></div>
      <div class="logo_grid3"></div>
      <div class="logo_grid4"></div>
```

File: lesson-1/farzad/styles.css [hunk 7]
```diff
@@ -0,0 +1,43 @@
+.logo {
+    display:inline-grid;
+    grid-template-columns:auto auto;
+    gap:5px;
+    width: 10vw;
+}
+
+.grid2 {
```
Comment (mehrzadnaini) [inline lesson-1/farzad/styles.css:8]: On `CSS` files it's better to sort your code based on usage 
e.g 
.grid1{
}
.grid2{
}
....

**Review by mehrzadnaini — CHANGES_REQUESTED** [review]

File: lesson-1/farzad/styles.css (hunk 7 above)
Comment (farzadabdollahi) [reply lesson-1/farzad/styles.css:8]: Ok i will correct it.thanks

**Review by mehrzadnaini — APPROVED** [review]

**Pull request merged**